csharp/src/Ice/Communicator.cs

```diff
@@ -194,7 +194,7 @@ internal CancellationToken CancellationToken
 
         internal TimeSpan IdleTimeout { get; }
         internal int IncomingFrameMaxSize { get; }
-        internal bool IsDisposed => _disposeTask != null;
+        internal bool IsDisposed => _destroyTask != null;
```

Comment:
Perhaps consider renaming to IsDestroyed?

Reply:
I prefer to keep IsDisposed since we use this property only for:
```
            lock (_mutex)
            {
                if (IsDisposed)
                {
                    throw new CommunicatorDisposedException();
                }
            ...
```